app/models/sql/ApplicationRow.scala

```diff
@@ -0,0 +1,104 @@
+package models.sql
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+import anorm.{Macro, RowParser}
+import cats.implicits.catsSyntaxTuple2Semigroupal
+import helper.Time
+import models.Application.Mandat
+import models.{Answer, Application}
+import serializers.DataModel
+
+final case class ApplicationRow(
+    id: UUID,
+    creationDate: ZonedDateTime,
+    creatorUserName: String,
+    creatorUserId: UUID,
+    subject: String,
+    description: String,
+    userInfos: Map[String, String],
+    invitedUsers: Map[UUID, String],
+    area: UUID,
+    irrelevant: Boolean,
+    answers: List[Answer] = List.empty[Answer],
+    internalId: Int = -1,
+    closed: Boolean = false,
+    seenByUserIds: List[UUID] = List.empty[UUID],
+    usefulness: Option[String] = Option.empty[String],
+    closedDate: Option[ZonedDateTime] = Option.empty[ZonedDateTime],
+    expertInvited: Boolean = false,
+    hasSelectedSubject: Boolean = false,
+    category: Option[String] = Option.empty[String],
+    files: Map[String, Long] = Map.empty[String, Long],
+    mandatType: Option[Application.Mandat.MandatType],
```

Comment:
Pour postgres c'est un `varchar`, que penses-tu de mettre la conversion dans `toApplication` plutôt ?
```suggestion
    mandatType: Option[String],
```